src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -153,9 +153,11 @@ export default {
                 : this.imageUrl;
         },
         getTime(datum) {
-            return datum
+            let dateTimeStr = datum
                 ? this.timeFormat.format(datum)
                 : this.time;
+            // Replace ISO "T" with a space to allow wrapping
+            return dateTimeStr ? dateTimeStr.replace("T"," ") : "";
```

Comment:
This is fine for now, but we need a more generic solution to this problem. Especially if they're expecting to see the `T` in the formatted output.
Since the `T` in ISO time indicates the beginning of the time element. Maybe instead of eliminating the `T` can we keep it and put a space before it instead? Like this

`2020-10-01 T22:50:49.897Z`

Reply:
The T is needed when the it's all smashed together, but the space makes it more readable and it gets wrapped to 2 lines in the thumb element - it's easier to parse without the T in that case.

Agree on the more generic solution. A better approach here I think would be to use proper time-formatting code that can handle a variety of time formats as input, and give me back separated date and time as two values. Wrapping the text using a space character is too brittle.

Reply:
I see your point.

Reply:
The problem, as I see it, is that we have conflating parsing raw telemetry data with formatting for human consumption in our API. These are two different problems, and they're not necessarily linked. Telemetry metadata should specify a "parser" for telemetry timestamps, not a formatter. Formatting for human consumption is a separate problem, and should be handled as such. For one, it needs to be context-sensitive. What's appropriate for a table doesn't necessarily work for a scale in a plot for example.